bigbluebutton-html5/client/main.jsx

```diff
@@ -37,6 +37,9 @@ function loadLib(libname, success, fail) {
 };
 
 Meteor.startup(() => {
+
+  localStorage.clear();
```

Comment:
I cleared local storage  in my solution because, if its not initially cleared, it doesn't load the default size if the page is reloaded.